elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -0,0 +1,113 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: clipboard;
+
+// Example local variable, using BEM format
+// --pfe-clipboard__region--Property: value;
+
+// This variable is global so that helper functions can reference it
+$LOCAL-VARIABLES: (
+  FontSize: 1rem,
+  FontWeight: pfe-var(font-weight--light),
+  Color: pfe-broadcasted(link),
+  Color--hover: pfe-broadcasted(link--hover),
+  Color--focus: pfe-broadcasted(link--focus),
+  text--success--Color: pfe-var(feedback--success),
+  icon--Width: 16px,
```

Comment:
```suggestion
  icon--Width: pfe-var(icon-size),
```

Reply:
It will give you `1em` but I think that makes sense since the text size is `16px`.  This should scale with the type.